src/components/modals/TransferPDNTModal/TransferPDNTModal.tsx

```diff
@@ -180,23 +204,57 @@ function TransferPDNTModal({
             ) : (
               <></>
             )}
+            {validatingAddress ? (
+              <Loader
+                size={80}
+                message={'Validating address, please wait...'}
+              />
+            ) : addressError ? (
+              <span
+                className="flex flex-row"
+                style={{
+                  gap: 10,
+                  alignItems: 'flex-start',
+                }}
+              >
+                <AlertTriangleIcon
+                  width={20}
+                  height={20}
+                  fill={'var(--accent)'}
+                />
+                <span
+                  className="text faded"
+                  style={{ textAlign: 'left', width: '90%' }}
+                >
+                  {addressError.message}
+                </span>
+              </span>
```

Comment:
can't we use the existing `valid` state variable?

Reply:
we want to show the error message if one is captured, this is just more granular

Reply:
added